_observing-your-data/alerting/comp-monitors.md

```diff
@@ -0,0 +1,633 @@
+                        "OkGceIgBYKBG2nHoyHAn|test1",
+                        "O0GceIgBYKBG2nHozHCW|test1"
+                    ],
+                    "nkjkj": [
+                        "OkGceIgBYKBG2nHoyHAn|test1",
+                        "O0GceIgBYKBG2nHozHCW|test1"
+                    ],
+                    "jknkjn": [
+                        "OkGceIgBYKBG2nHoyHAn|test1",
+                        "O0GceIgBYKBG2nHozHCW|test1"
+                    ]
+                }],
+                "error": null
+            },
+            "trigger_results": {
+                "NC3Dd4cBCDCIfBYtViLI": {
+                    "name": "njkkj",
+                    "triggeredDocs": [
+                        "OkGceIgBYKBG2nHoyHAn|test1",
+                        "O0GceIgBYKBG2nHozHCW|test1"
+                    ],
+                    "action_results": {},
+                    "error": null
+                }
+            }
+        },
+        {
+            "monitor_name": "test triggers 2",
+            "period_start": 1685650668501,
+            "period_end": 1685650728501,
+            "error": null,
+            "input_results": {
+                "results": [{
+                    "bhjh": [
+                        "PEGceIgBYKBG2nHo1HCw|test",
+                        "PUGceIgBYKBG2nHo3HA8|test"
+                    ],
+                    "nkjkj": [
+                        "PEGceIgBYKBG2nHo1HCw|test",
+                        "PUGceIgBYKBG2nHo3HA8|test"
+                    ],
+                    "jknkjn": [
+                        "PEGceIgBYKBG2nHo1HCw|test",
+                        "PUGceIgBYKBG2nHo3HA8|test"
+                    ]
+                }],
+                "error": null
+            },
+            "trigger_results": {
+                "NC3Dd4cBCDCIfBYtViLI": {
+                    "name": "njkkj",
+                    "triggeredDocs": [
+                        "PEGceIgBYKBG2nHo1HCw|test",
+                        "PUGceIgBYKBG2nHo3HA8|test"
+                    ],
+                    "action_results": {},
+                    "error": null
+                }
+            }
+        }
+    ],
+    "execution_start_time": "2023-06-01T20:18:48.511874Z",
+    "execution_end_time": "2023-06-01T20:18:53.682405Z",
+    "error": null
+}
+```
+
+
+### Acknowledge Chained Alerts
+
+This API provides a list of chained alerts belonging to a specific workflow. 
+
+```json
+POST _plugins/_alerting/workflows/<workflow_id>/_acknowledge/alerts
+{
+    "alerts": ["eQURa3gBKo1jAh6qUo49"]
+}
+```
+{% include copy-curl.html %}
+
+#### Request fields
+
+| Field | Type | Description |
+| :--- | :--- | :--- |
+| `alerts` | Array | A list of alerts by Id. The results include alerts that are acknowledged by the system as well as alerts not recognized by the system. |
+
+#### Example response
+
+```json
+{
+    "success": [
+    "eQURa3gBKo1jAh6qUo49"
+    ],
+    "failed": []
+}
+```
+
+
+### Get Chained Alerts API
+
+This API 
```

Comment:
Returns chained alerts generated by workflows. If workflowIds param is mentioned, it returns alerts created by specific workflow
getAssociatedAlerts  flag if set to true returns the AUDIT state alerts created by delegate monitors which matched the chained alert trigger condition to create to a chained alert. It’s an optional flag which defaults to false